src/components/expanded-state/unique-token/NFTBriefTokenInfoRow.tsx

```diff
@@ -67,27 +51,6 @@ export default function NFTBriefTokenInfoRow({
 
   const currentPrice = asset?.currentPrice ?? listingValue;
 
-  useEffect(() => {
```

Comment:
all nfts now come from simplehash which includes floor price, so we no longer need this opensea fallback